conf/application.conf

```diff
@@ -51,6 +51,8 @@ play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 play.modules.enabled += "config.Module"
+play.additional.module = "config.TransitionModule"
+
 
```

Comment:
Hey, would you mind adding the variable to override?
`play.modules.enabled += ${play.additional.module}`